crates/bevy_picking/src/mesh_picking/ray_cast/intersections.rs

```diff
@@ -17,9 +17,9 @@ pub struct RayMeshHit {
     /// The distance from the ray origin to the intersection point.
     pub distance: f32,
     /// The vertices of the triangle that was hit.
```

Comment:
I'm curious if these were defined as `Option` so that the API wouldn't have to change if/when support for `PrimitiveTopology` other than `TriangleList` was added.

Reply:
Good point. But wouldn't `barycentric_coords: Vec3` also have to be an `Option` in case on non-triangles?
Anyway, let's keep this change separate. I will revert this change and create separate PR

Reply:
@aloucks: Do you mind if I close this conversation?

Reply:
go for it 👍